redact/redact.go

```diff
@@ -77,6 +77,10 @@ func Apply(redactions []*Redact, w io.Writer, r io.Reader) error {
 // TODO(mkcp): Speed improvement & out of memory error: JSON responses can be really big, so we're going to have to
 //  chunk extremely large strings down.
 func String(result string, redactions []*Redact) (string, error) {
+	if len(redactions) == 0 {
```

Comment:
FYI, due to the benchmarks added in #198 I found that, on my machine, when there are no redactions, this short-circuiting runs in about 2-3% of the time than without the short circuit.